packages/dependency-extraction-webpack-plugin/lib/index.js

```diff
@@ -214,23 +214,26 @@ class DependencyExtractionWebpackPlugin {
 					}
 				}
 
-				const runtimeChunk = entrypoint.getRuntimeChunk();
+				const entrypointChunk = entrypoint.chunks.find(
```

Comment:
This is a method for finding the entrypoint chunk that I found through trial and error, but I don't now the webpack internals enough to say if it is reliable.

Reply:
I'm doing my best to follow up on this but don't lot of time to spare, so thanks for your patience.

This does look questionable and I don't have a deep understanding of webpack internals either (they're mysterious).

It looks like webpack 5 introduces `Entrypoint.getEntrypointChunk()` which would be helpful here 🙄 

https://github.com/webpack/webpack/blob/ec8e1bb0b3b8203818741f731fdf654e1583b2d4/lib/Entrypoint.js#L80-L87

I'm tempted to wait for #26382.

Reply:
I was also drawn to `Entrypoint.getEntrypointChunk()` until I realised that it wasn't available in Webpack 4, so I totally agree that waiting for Webpack 5 support makes sense instead of praying that the weird hack that seems to work doesn't break in weird and wonderful ways :)

Reply:
It might take another couple of weeks to land webpack 5. Once it happens we should drop support for webpack 4 to simplify this plugin.

Reply:
We are sort out our second attempt to upgrade webpack from v4 to v5 in https://github.com/WordPress/gutenberg/pull/33818. The remaining question would be if we are going to keep support for both versions v4 and v5 of webpack on the plugin.

Reply:
Is there anything I can do to help get this PR going again?